### ISSUE TITLE
[cetic/nifi] correct the unintuitive Service

### ISSUE THREAD

**Issue by alexnuttinck** [cetic/helm-nifi#17]
**Is your feature request related to a problem? Please describe.**

The current values.yaml is unintuitive to setup the service part. (#7 , #16)

**Describe the solution you'd like**

Refactor the Service part: maybe rename the `loadbalancer` into `ui` and add some comments into the `values.yaml`

**Comment by alexnuttinck** [comment]
The [README.md](https://github.com/cetic/helm-nifi#configure-the-chart) should be rework too.

**Comment by alexnuttinck** [comment]
Old service definition:
```
service:
  headless:
    type: ClusterIP
  loadBalancer:
    enabled: true
    type: LoadBalancer
    httpPort: 80
    httpsPort: 443
    annotations: {}
```

New service definition:

```
# headless service
headless:
  type: ClusterIP
  annotations: 
    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"

# ui service
service:
  type: LoadBalancer
  httpPort: 80
  httpsPort: 443
  annotations: {}
```

**Comment by alexnuttinck** [comment]
merged in [v0.3.0](https://github.com/cetic/helm-nifi/releases/tag/0.3.0).
